src/mcts/params.cc

```diff
@@ -275,6 +275,25 @@ const OptionId SearchParams::kSolidTreeThresholdId{
     "solid-tree-threshold", "SolidTreeThreshold",
     "Only nodes with at least this number of visits will be considered for "
     "solidification for improved cache locality."};
+const OptionId SearchParams::kTaskWorkersPerSearchWorkerId{
+    "task-workers", "TaskWorkers",
+    "The number of task workers to use to help the search worker."};
+const OptionId SearchParams::kMinimWorkSizeForProcessingId{
```

Comment:
Why is it `Minim` while others are `Minimum`? :)

Reply:
Typos :)

Reply:
Fixed in 1490.